### PR TITLE
Reduce words

### PR DESCRIPTION
**Pull request by knoxcard**
Reduce words, better structure.

- [x] Ready for review
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

### CODE REVIEW

**Comment by lili2311** [comment]
👋 Please could you squash to 1 commit and change it to follow the format from teh Contibuting guide? https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md

File: README.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ docker run -it
 
 ## Badge
 
-You can put a badge on your README showing your package has no known security holes. This will show your users you care about security, and tell them that they should care too.
+Make users feel more confident in using your website by adding your Snyk badge!
```
Comment (racheyfi) [inline README.md]: Help your users feel more confident when using your website by adding your Snyk badge!

**Comment by lili2311** [comment]
@knoxcard will you have a chance to address the comments soon?

File: README.md (hunk 1 above)
Comment (knoxcard) [reply README.md]: Love it!

**Comment by knoxcard** [comment]
Think I just addressed the comment sections by "resolving".  Anything else I need to do?

**Comment by lili2311** [comment]
> Think I just addressed the comment sections by "resolving". Anything else I need to do?
I will take your commits and create a PR that can can have passing tests as you won't have access for the snyk org token to make all tests pass. Will let you know once done.

**Comment by lili2311** [comment]
https://github.com/snyk/snyk/pull/530 >> closing this one, your commit is in this PR

**Pull request closed without merging**